news/2023/2023-02-15-featured-artist-updates-mwc7k.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "osu! originals - osu!mania 7K World Cup 2023"
+date: 2023-02-15 11:00:00 +0000
+---
+
+As the [osu!mania World Cup 7K 2023](/wiki/Tournaments/MWC/2023_7K) approaches its climax, take a moment to explore some of the **5** brand-new songs created specifically for the tournament!
+
+![](/wiki/shared/news/banners/featured-artist.jpg)
+
+But first, some context.
+
+Our Featured Artist program has expanded over the years, and we've only recently started dipping our feet into the depths of *original song commissions* (which we call [osu! originals](/wiki/osu!_originals)). You might've seen a few osu! originals already via Featured Artist announcements or the recent [Twin Trials contest](https://osu.ppy.sh/home/news/2023-02-01-twin-trials-contest-beatmapping-phase).
```

Comment:
hmm, i feel like this is dismissing the originals we've had created over the years a bit by saying "only recently". some of the first featured artists were actually brought on with originals as their initial contribution (IAHN for example). we've also had some amazing landino tracks, cysmix tracks, as well as other artists over the years. not sure how to change the wording in a way that would hint at this, but maybe would be nice to?

Reply:
this is referring to original commissions for tournaments, but i guess the wording doesn't convey that properly

Reply:
maybe something like this

```suggestion
Our Featured Artist program has expanded over the years, and we've only recently started dipping our feet into the depths of *original tournament track commissions* (also known under the [osu! originals](/wiki/osu!_originals) label). You might've seen a few osu! originals already via Featured Artist announcements over the years, or perhaps from the recent [Twin Trials contest](https://osu.ppy.sh/home/news/2023-02-01-twin-trials-contest-beatmapping-phase).
```

Reply:
Good enough I think!